ac_dc/tests/test_govt_anonymization.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+import anonymization
+
+
+@pytest.mark.parametrize("sentence,lang_id,expected", [
+    ("hello world", "en", "hello world"),
+    ("my government id is 123 123 123", "en", "my government id is 111 111 111"),
```

Comment:
@ontocord can you help add unit tests here :)? at the same time we can sanity check that the regexes are doing what we're expecting